daml-script/daml3/Daml/Script/Internal.daml

```diff
@@ -162,3 +163,7 @@ fromLedgerValue = dangerousCast
 
 toLedgerValue : a -> LedgerValue
 toLedgerValue = dangerousCast
+
+data Disclosure = Disclosure with
+    blob: Text
+  deriving (Eq, Ord)
```

Comment:
Newline here, and maybe shouldn't go in `Internal`, this is just the most generised Question logic

Reply:
Where should I put it then ?

Reply:
Not sure, tempted to say Submit, or a new module